### ISSUE TITLE
context_encoder

### ISSUE THREAD

**Issue by kareem337** [eriklindernoren/PyTorch-GAN#168]
I would like to know if i can make the discriminator classify multiple classes using softmax just like ccgan code.
